pkg/objectupdate/rte/rte.go

```diff
@@ -136,6 +136,13 @@ func DaemonSetArgs(ds *appsv1.DaemonSet, conf nropv1.NodeGroupConfig) error {
 		return fmt.Errorf("cannot modify the arguments for container %s", cnt.Name)
 	}
 
+	rtePauseEnabled := isRTEPauseEnabled(&conf)
+	klog.V(2).InfoS("DaemonSet update: RTEPause status", "daemonset", ds.Name, "enabled", rtePauseEnabled)
+	if rtePauseEnabled {
+		klog.V(4).InfoS("DaemonSet update: disable publishing NRTs updates")
+		flags.SetToggle("--no-publish")
+	}
```

Comment:
if `rtePauseEnabled` is false, we need to remove the toggle if present. Otherwise, when added, it will be always kept here